hector_mapping/launch/mapping_default.launch

```diff
@@ -41,6 +41,9 @@
     
     <param name="scan_subscriber_queue_size" value="$(arg scan_subscriber_queue_size)"/>
     <param name="scan_topic" value="$(arg scan_topic)"/>
+
+    <!-- Uncomment the following if there are any laser scanners that are static w.r.t. the robot's base_frame. If so, this node caches these static transforms at node startup, preventing repetitive tf tree lookups in unreliable networks -->
+    <!-- <rosparam param="static_laser_frame_ids"> ["example_laser_frame1", "example_laser_frame2"] </rosparam> -->
```

Comment:
I added this as a comment in case other users want to benefit from caching static tf tree lookup. Note that we have to use `rosparam`, as [it allows to parse multiple inputs](https://answers.ros.org/question/291984/reading-vector-from-launch-file-does-not-work-in-indigo/) (in case the user has multiple lidars on the robot), whereas `param` doesn't

Reply:
If this is commented or doesn't have any input frames, then no transforms will be cached.
I wanted to keep this comment to serve as an example for people to use, as most people are not familiar with passing vectors through `rosparam`